content/messaging/agent-chat-api/rtm-reference/index.mdx

```diff
@@ -425,39 +425,64 @@ The **Message** event contains text message to other chat users.
-| `elements.buttons.text`          | optional  | Required only if `elements.buttons` is present.                     |
-| `elements.buttons.type`          | optional  | Required only if both `elements.buttons` and `elements.buttons.value` are present. Possible values: `webview`, `message`, `url`, `phone`. [More...](/extending-ui/extending-chat-widget/rich-messages/#getting-started)  |
-| `elements.buttons.user_ids`      | optional  | Required only if `elements.buttons` is present. Describes users that sent the postback with `"toggled": true`. |
-| `elements.buttons.value`         | optional  | Required only if both `elements.buttons` and `elements.buttons.type` are present.  |
-| `elements.buttons.webview_height`| optional  | Possible values: `compact`, `full`, `tall`                                                                    |
-| `elements.image`                 | optional  | `image` properties are optional: `name`, `url`, `content_type`, `size`, `width`, `height`. |
-| `elements.subtitle`              | optional  |                                                                     |
-| `elements.title`                 | optional  |                                                                     |
-| `properties`                     | optional  | The [properties](#properties) object                                |
+Rich message (RM) event contains a rich message data structure. [Read more](/extending-ui/extending-chat-widget/rich-messages/) about rich messages.
+
+### Request
+
+| Field                            | Required | Data type | Notes                                                                                             |
+| -------------------------------- | -------- | --------- | ------------------------------------------------------------------------------------------------- |
+| `custom_id`                      | no       | `string`  | You can give your RM a custom ID.                                                                 |
+| `type`                           | yes      | `string`  | Event type: `rich_message`                                                          |
+| `recipients`                     | yes      | `string`  | Those who can display the rich message: `all` (default) or `agents`                               |
+| `properties`                     | no       | `object`  | The [properties](#properties) data structure                                                              |
+| `template_id`                    | yes      | `string`  | Defines how every Rich Message will be presented. Values: `card` or `quick_replies`.              |
+| `elements`                       | no       | `array`   | Can contain up to 11 `element` objects.                                                           |
```

Comment:
Should be `10`. cc @klarzynskik for CW docs (you said that you're going to correct that).